src/mito2/src/flush.rs

```diff
@@ -88,6 +88,9 @@ pub struct WriteBufferManagerImpl {
     memory_used: AtomicUsize,
     /// Memory that hasn't been scheduled to free (e.g. used by mutable memtables).
     memory_active: AtomicUsize,
+    /// Optional notifier.
+    /// The manager can wake up the worker once we free the write buffer.
+    notifier: Option<watch::Sender<()>>,
```

Comment:
When would this optional notifier be set and consumed?

Reply:
The worker group will set it.